Makefile

```diff
@@ -34,12 +34,17 @@ $(info $(shell printf "%-21s = %-10s\n" "KUBE_CONTROLLERS_VER" $(KUBE_CONTROLLER
 $(info $(shell printf "%-21s = %-10s\n" "TYPHA_VER" $(TYPHA_VER)))
 ##############################################################################
 
-serve:
-	docker run --rm -ti -e JEKYLL_UID=`id -u` -p 4000:4000 -v $$PWD:/srv/jekyll jekyll/jekyll:$(JEKYLL_VERSION) jekyll serve --incremental $(CONFIG)
+serve: helm
+	# We have to override JEKYLL_DOCKER_TAG which is usually set to 'pages'. 
+	# When set to 'pages', jekyll starts in safe mode which means it will not
+	# load any plugins. Since we're no longer running in github-pages, but would
+	# like to use a docker image that comes preloaded with all the github-pages plugins,
+	# its ok to override this variable.
+	docker run -e JEKYLL_DOCKER_TAG="" -v $$PWD/bin/helm:/usr/local/bin/helm:ro --rm -ti -e JEKYLL_UID=`id -u` -p 4000:4000 -v $$PWD:/srv/jekyll jekyll/jekyll:$(JEKYLL_VERSION) jekyll serve --incremental $(CONFIG)
```

Comment:
Should we just change the default from `pages` to something else?

Reply:
We could, but the `pages` image comes preloaded with all the jekyll plugins we need. So if we use another image that just has jekyll, we would need to `gem install github-pages` first (which takes a lot of time) _on every single run_.  I think the correct option would be to build and publish our own image to dockerhub, but I obviously don't want to add another thing to maintain